### DISCUSSION TITLE
Get Github Achievements Easy - Step By Step

### DISCUSSION THREAD

**Discussion by YasinDehfuli** [community/community#62495] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

If You need help to get GitHub achievements Step by Step by image screenshots & Descriptions.

You Can Use This Project to get achievements easy:

**English**: [English Guide](https://github.com/4xmen/Get-Github-Achievements)

**Persian**: [Farsi Guide](https://github.com/4xmen/Get-Github-Achievements/blob/main/README.fa.md)

**German** : [Deutsche Guide](https://github.com/4xmen/Get-Github-Achievements/blob/main/README.de.md)

**French** : [French Guide](https://github.com/4xmen/Get-Github-Achievements/blob/main/README.fr.md)

**Russian**: [Russian Guide](https://github.com/4xmen/Get-Github-Achievements/blob/main/README.ru.md)

**Turkish**: [Turkish Guide](https://github.com/4xmen/Get-Github-Achievements/blob/main/README.tr.md)

**Dutch**  : [Dutch Guide](https://github.com/4xmen/Get-Github-Achievements/blob/main/README.nl.md)

................................................................................................

If You can help, To translate project to other languages. Join us as a contributor.
Get a Fork and start translate project to other languages.

**Comment by AliYa-chen** [comment]
Hello, I am a Github user from China. Can I help translate it into Chinese?

**Reply by YasinDehfuli** [reply]
Hi @AbrnYa, yes, of course, I appreciate it. If you need help to translate, there are some ways to contact me.

**Reply by AliYa-chen** [reply]
That's very kind of you. I also stumbled upon it and did my best to help you translate it into Chinese!

**Comment by Vicwxy9** [comment]
https://github.com/4xmen/Get-Github-Achievements follow this link and you will get the guided steps for it

**Comment by queenofcorgis** [comment]
Hi @YasinDehfuli, we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.

As a reminder, GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers.

**Reply by YasinDehfuli** [reply]
Hi @queenofcorgis, I'm sorry to hear that, it's because of my mistake, and thank you for your message :)
